src/airflow/dags/kcca_measurements.py

```diff
@@ -37,11 +37,12 @@ def transform(data: pd.DataFrame):
     def send_to_api(data: pd.DataFrame):
         from airqo_etl_utils.kcca_utils import KccaUtils
         from airqo_etl_utils.airqo_api import AirQoApi
+        from airqo_etl_utils.constants import Tenant
 
         kcca_data = KccaUtils.transform_data_for_api(data)
 
         airqo_api = AirQoApi()
-        airqo_api.save_events(measurements=kcca_data, tenant="kcca")
+        airqo_api.save_events(measurements=kcca_data, tenant=Tenant.KCCA)
```

Comment:
tenant is still KCCA here. Perhaps I could be missing something. @NoahNsimbe 

Based on my understanding at the moment.....

**Options/alternatives:**

1. We can still send to the KCCA DB (tenant) while also sending the KCCA events to AIRQO with the usage of the NETWORK field. Good for redundancy.
2. OR, we can just send everything to one DB (tenant=AIRQO) and completely eliminate the usage of other tenants during insertion. This will require the abandonment of some util files here?

Thanks.

Reply:
This has no effect. it is used to identify the network. @Baalmart.